pkg/syndesis/action/install.go

```diff
@@ -87,3 +123,57 @@ func newSyndesisServiceAccount() *corev1.ServiceAccount {
 
 	return &sa
 }
+
+
+func installSyndesisRoute(syndesis *v1alpha1.Syndesis, objects []runtime.Object) (*v1.Route, error) {
+	route, err := findSyndesisRoute(objects)
+	if err != nil {
+		return nil, err
+	}
+
+	setNamespaceAndOwnerReference(route, syndesis)
+	// We don't replace the route if already present, to let Openshift generate its host
+	err = sdk.Create(route)
+	if err != nil && !k8serrors.IsAlreadyExists(err) {
+		return nil, err
+	}
+
+	if route.Spec.Host != "" {
+		return route, nil
```

Comment:
Wouldnt it be better to always pickup the host name from the generated route (like below), just to be sure ?